core/cap-0046-02.md

```diff
@@ -199,29 +157,34 @@ functionality to support them.
 The following host functions are provided to instantiate contracts:
```

Comment:
...to instantiate contracts and upload Wasm:

Reply:
Done